### ISSUE TITLE
Temp fix for the white screen on boot

### ISSUE THREAD

**Issue by xXxTheDarkprogramerxXx** [xXxTheDarkprogramerxXx/PS3Tools#217]
Temp fix for the white screen on boot 
https://github.com/xXxTheDarkprogramerxXx/PS3Tools/pull/215

_Originally posted by @GarnetSunset in https://github.com/xXxTheDarkprogramerxXx/PS3Tools/issues/150#issuecomment-805789175_

**Comment by xXxTheDarkprogramerxXx** [comment]
Hi so the disabling of the boot screen is fine however I am on the latest version of windows and mine runs fine.

Might be something else.

Mind giving me you're pc specs.
P.s. do you have a sound card ?

**Comment by ghost** [comment]
I do not have a sound card, and i cant find logs for your program, it appears the video file just never starts, when disabling the video file by default it seems work fine.

**Comment by xXxTheDarkprogramerxXx** [comment]
Id rather add a check (if possible) for the bootscreen than just remove it completely as thats not a fix for rhe issue at hand

**Comment by ghost** [comment]
why not a config file where it can be set prior to running?

**Comment by xXxTheDarkprogramerxXx** [comment]
> why not a config file where it can be set prior to running?

Cause most people are not very computer literate so we need to make it as seamless as possible

**Comment by ghost** [comment]
That's true, I'm not sure how to debug it, as A. I'm not a C based programmer and B. there isn't a debug log i can find

**Comment by Goon3r** [comment]
Can confirm the following worked to fix the white screen for me.

`
<setting name="EnableBootScreen" serializeAs="String">
    <value>False</value>
</setting>
`

Note, i'm on a Mac using windows 10 via parallels and the only way i was able to get this to run was to use the `setup.exe` file from the v1.0 release and then manually trackdown the exe config file and update it.

Some further details:

- Running the lastest debug release and executing the .exe directly lead to app closing after a couple of seconds and if quickly clicking through initial options it also triggered the ps2.mp4 file not found
- Running the lasted debug release .application file lead to lots of `*.deploy` files not found.
- Running the v1 .application file installed v1.0.0.3 succesfully but couldn't update
- Running the v1 setup.exe file allowed me to install the lasted version 1.0.0.28 and through tracking down the source exe from the app reference i was able to manually adjust the config file and bypass the whitescreen.

This is my first install, and i haven't had a chance to test functionality as yet.
